question4.py

```diff
@@ -0,0 +1,23 @@
+# Question 4
+
+
+def fibo(n):
+    if n<=1:
+        return 1
+    else:
+        return fibo(n-1) + fibo(n-2)
+
+terms = input("Enter the nth term: ")
```

Comment:
Enter the input as 60 or 90, if the code stucks then optimize the code using memorization technique which is mentioned in the doc.

Reply:
Done